utils/erc-20/erc-20.test.ts

```diff
@@ -0,0 +1,8 @@
+import { getERC20Data } from '.';
+
+describe('ERC-20', () => {
+  it('Should be a true because are different', () => {
+    /* test to see what value it return*/
+    console.log(getERC20Data(97, '0xfF6FD90A470Aaa0c1B8A54681746b07AcdFedc9B'));
```

Comment:
test that it gives the same token if u pass symbol or address on the chainId. and diff if it is not a different chainId